addons/account_followup/account_followup.py

```diff
@@ -1,125 +1,90 @@
 
-Best Regards,
-""",
-    'email_template_id': _get_default_template,
-    }
+    name = fields.Char('Follow-Up Action', required=True)
+    sequence = fields.Integer('Sequence', help="Gives the sequence order when displaying a list of follow-up lines.")
+    delay = fields.Integer('Due Days', help="The number of days after the due date of the invoice to wait before sending the reminder.      Could be negative if you want to send a polite alert beforehand.", required=True)
+    followup_id = fields.Many2one('account_followup.followup', 'Follow Ups', required=True, ondelete="cascade")
+    description = fields.Text('Printed Message', default=_get_default_description, translate=True)
+    send_email = fields.Boolean('Send an Email', default=True, help="When processing, it will send an email")
+    send_letter = fields.Boolean('Send a Letter', default=True, help="When processing, it will print a letter")
+    manual_action = fields.Boolean(default=False, help="When processing, it will set the manual action to be taken for that customer. ")
+    manual_action_note = fields.Text('Action To Do', placeholder="e.g. Give a phone call, check with others , ...")
+    manual_action_responsible_id = fields.Many2one('res.users', 'Assign a Responsible', ondelete='set null')
+    email_template_id = fields.Many2one('mail.template', 'Email Template', default=_get_default_template, ondelete='set null')
 
+    _order = 'delay'
+    _sql_constraints = [('days_uniq', 'unique(followup_id, delay)', 'Days of the follow-up levels must be different')]
 
-    def _check_description(self, cr, uid, ids, context=None):
-        for line in self.browse(cr, uid, ids, context=context):
+    @api.one
+    @api.constrains('partner_name', 'date', 'user_signature', 'company_name')
+    def _check_description(self):
+        for line in self:
             if line.description:
                 try:
-                    line.description % {'partner_name': '', 'date':'', 'user_signature': '', 'company_name': ''}
+                    line.description % {'partner_name': '', 'date': '', 'user_signature': '', 'company_name': ''}
                 except:
                     return False
-        return True
+        raise ValidationError(_('Your description is invalid, use the right legend or %% if you want to use the percent character.'))
 
-    _constraints = [
-        (_check_description, 'Your description is invalid, use the right legend or %% if you want to use the percent character.', ['description']),
-    ]
 
+class AccountMoveLine(models.Model):
 
-class account_move_line(osv.osv):
-
-    def _get_result(self, cr, uid, ids, name, arg, context=None):
+    @api.multi
+    def _get_result(self):
         res = {}
-        for aml in self.browse(cr, uid, ids, context=context):
+        for aml in self:
             res[aml.id] = aml.debit - aml.credit
```

Comment:
Use new API style to convert the compute method !!!
Ex:
self.field_name = field_value